posts/2022/10-11-graphql_subscription_setup.md

```diff
@@ -0,0 +1,246 @@
+==title==
+GrapQL subscriptions with Absinthe - the set-up
+
+==author==
+Cornelia Kelinske
+
+==description==
+This is part 1 of a 3-part series on setting up and testing GraphQL subscriptions with Absinthe.
+
+==tags==
+coding, elixir, graphql, absinthe, subscriptions
```

Comment:
I don't like the coding tag... it is very "code monkey" feeling.

Is there an alternative tag?